3rdparty/fastcv/hal/fastcv_hal_imgproc.cpp

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "fastcv_hal_imgproc.hpp"
+#include "fastcv_hal_utils.hpp"
+#include <opencv2/core/base.hpp>
+#include <opencv2/core/utils/logger.hpp>
+#include <opencv2/imgproc.hpp>
+
+extern bool isInitialized;
```

Comment:
This can be moved to `fastcv_hal_utils.hpp`

Reply:
will update in next PR